src/BannerWithLink/index.tsx

```diff
@@ -1,18 +1,16 @@
-import React, { useState } from 'react';
-import { LinkButton } from '@guardian/source-react-components';
+import React from 'react';
+import { StyleableBannerWithLink } from '../StyleableBannerWithLink';
 import type { TrackClick } from '../utils/tracking';
-import { BannerColorStyles } from '../styles/colorData';
-import { BannerCloseButton } from '../components/BannerCloseButton';
-import { selfServeStyles } from '../styles/bannerCommon';
-import { canRender, COMPONENT_NAME } from './canRender';
 
+import { canRender, COMPONENT_NAME } from './canRender';
 export { COMPONENT_NAME };
 
 export type BrazeMessageProps = {
     ophanComponentId?: string;
     header?: string;
     body?: string;
     boldText?: string;
+    highlight?: string;
```

Comment:
It feels slightly odd that this has been added alongside `boldText`. Below it looks like we'll use `boldText` and pass it to `StyleableBannerWithLink` as highlight. So should it be part of the interface of this component?

Reply:
Agreed. Will remove it. Hopefully we can persuade Marketing to stop using this banner in the near future, then remove it from the repo.